src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -320,6 +321,7 @@ export default class ArrayField extends React.Component {
                               className="usa-button-secondary float-right"
                               type="button"
                               onClick={() => this.handleRemove(index)}
+                              aria-label={`Remove ${title}`}
```

Comment:
@Mottie I'm going to reverse my field on adding `aria-label` to these Remove buttons. When I listened to them on localhost, it was more confusing to have the longer label in some places and not in others. Let's make all the Remove buttons just say Remove as before.

Apologies for asking for a second round of changes. No further review needed from me after this.